### PR TITLE
Score backend

### LINKED ISSUE #319: [5] Retrieving the score for the color from the backend
A score which is shows how well the users is doing needs to be retrieved from the API.

### PR DESCRIPTION
**Pull request by MaxVanDeursen**
In this PR, a score attribute is added to the node object in the
TriblerChain community. This score is a number between 0 and 1,
indicating the score of the node in the network. Although this is now
calculated with an easy method, this can be adjusted to any other
calculation in the future.

Fixes #319

### CODE REVIEW

File: Tribler/community/triblerchain/score.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    :param node: the node for which the score has to be calculated
+    :return: a number between 0 and 1 which represents the score of the node.
+    """
+    balance = node["total_up"] - node["total_down"]
+    if balance < -BOUNDARY_VALUE:
+        return 0
+    elif balance > BOUNDARY_VALUE:
+        return 1
+    else:
+        return (balance + BOUNDARY_VALUE) / (BOUNDARY_VALUE + BOUNDARY_VALUE)
```
Comment (TimSpeelman) [inline Tribler/community/triblerchain/score.py]: Alternative: ```0.5 + balance / (2 * BOUNDARY_VALUE)```

File: Tribler/community/triblerchain/community.py [hunk 2]
```diff
@@ -237,6 +238,10 @@ def get_graph(self, public_key=None, neighbor_level=1):
 
         return_nodes = nodes.values()
         return_edges = []
+
+        for node in return_nodes:
+            node.update({"score": calculate_score(node)})
```
Comment (vandenheuvel) [inline Tribler/community/triblerchain/community.py]: Calling update is not necessary here, why not just `node["score"] = calculate_score(node)`?

File: Tribler/community/triblerchain/score.py (hunk 1 above)
Comment (MaxVanDeursen) [inline Tribler/community/triblerchain/score.py]: I think as it is now, the readability is improved. It shows that it comes from the calculation of a linear calculation.

File: Tribler/community/triblerchain/score.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+BOUNDARY_VALUE = pow(10, 9)
```
Comment (vandenheuvel) [inline Tribler/community/triblerchain/score.py]: Please add a module docstring.

File: Tribler/community/triblerchain/community.py (hunk 2 above)
Comment (MaxVanDeursen) [inline Tribler/community/triblerchain/community.py]: Good call!

**Pull request merged**